docs/fr/docs/index.md

```diff
@@ -75,48 +74,49 @@ The key features are:
 
 ---
 
-"_**Netflix** is pleased to announce the open-source release of our **crisis management** orchestration framework: **Dispatch**! [built with **FastAPI**]_"
+"_**Netflix** a le plaisir d'annoncer la sortie open source de notre structure d'orchestration de **gestion de crise** : **Dispatch** ! [construit avec **FastAPI**]_"
 
 <div style="text-align: right; margin-right: 10%;">Kevin Glisson, Marc Vilanova, Forest Monsen - <strong>Netflix</strong> <a href="https://netflixtechblog.com/introducing-dispatch-da4b8a2a8072" target="_blank"><small>(ref)</small></a></div>
 
 ---
-
-"_I’m over the moon excited about **FastAPI**. It’s so fun!_"
+"_Je suis sur la lune enthousiasmé par **FastAPI**. C'est tellement amusant !_"
```

Comment:
Traduction de *over the moon excited* assez complexe. Je propose *hyper enthousiaste*, le sens est très similaire je trouve. 

```suggestion
"_Je suis hyper enthousiaste par **FastAPI**. C'est tellement amusant !_"
```

Reply:
Je suis enchanté par **FastAPI**.